docs/interpreter/jdbc.md

```diff
@@ -507,8 +507,9 @@ Here are some examples you can refer to. Including the below connectors, you can
     <td>hive_password</td>
   </tr>
   <tr>
-    <td>hive.proxy.user</td>
-    <td>true or false</td>
+    <td>default.proxy.user.property</td>
+    <td>hive.server2.proxy.user</td>
+  </tr>
 </table>
 
 Connection to Hive JDBC with a proxy user can be disabled with `hive.proxy.user` property (set to true by default)
```

Comment:
I was looking at http://zeppelin.apache.org/docs/0.7.1/interpreter/jdbc.html#apache-hive, `hive.proxy.user` hasn't made it to the doc section, so I think in this case no need to add migration steps. However, I have added `logger.warn` as well.